common/lib/xmodule/xmodule/js/src/capa/schematic.js

```diff
@@ -3919,49 +3924,56 @@ schematic = (function() {
 	    var tool = this.tools[tname];
 
 	    if (tool != undefined) {
-		tool.style.opacity = which ? 1.0 : 0.2;
+        tool.removeAttribute('disabled');
 		tool.enabled = which;
 
 		// if disabling tool, remove border and tip
 		if (!which) {
-		    tool.style.borderColor = background_style;
+		    // tool.style.borderColor = background_style;
 		    tool.sch.message('');
+            tool.setAttribute('disabled', 'true');
 		}
 	    }
 	}
 
 	// highlight tool button by turning on border, changing background
 	function tool_enter(event) {
 	    if (!event) event = window.event;
-	    var tool = (window.event) ? event.srcElement : event.target;
-
-	    if (tool.enabled) {
-		tool.style.borderColor = normal_style;
+        var tool = event.target;
+        if (event.target.tagName.toLowerCase() == 'img' || event.target.tagName.toLowerCase() == 'span') {
+            tool = event.target.parentNode;
+        }
+        if (tool.enabled) {
 		tool.sch.message(tool.tip);
-		tool.opacity = 1.0;
 	    }
+        event.stopPropagation();
 	}
 
 	// unhighlight tool button by turning off border, reverting to normal background
 	function tool_leave(event) {
 	    if (!event) event = window.event;
-	    var tool = (window.event) ? event.srcElement : event.target;
-
+        var tool = event.target;
+        if (event.target.tagName.toLowerCase() == 'img' || event.target.tagName.toLowerCase() == 'span') {
```

Comment:
Previously these were images with JavaScript event listeners. No role or button so they weren't accessible to anything other than a mouse. I wrapped the images in a button to place it in the tab order and make it accessible. The span contains the text - either `.sr` or otherwise - within the buttons for additional context. The ultimate goal is to get the button element, which is a bit lengthy with vanilla JavaScript, but that's basically what this is doing.